### DISCUSSION TITLE
VirtualizedScrollPane + InlineCssTextArea scroll to absolute bottom after text appended

### DISCUSSION THREAD

**Discussion by noosphere888** [FXMisc/RichTextFX#1207] [Q&A] [answered]
Hello,

Is there a to automatically scroll to absolute bottom after text is appended to _InlineCssTextArea_ with _VirtualizedScrollPane_? Am making a logs box component that adds lines of text and trying to figure out a good way to automatically scroll to absolute bottom.

A basic ScrollPane lets you do this with:
`scrollPane.setVvalue(1.0);`
Is there something similar with VirtualizedScrollPane?  Maybe a way to force with a method like _scrollYBy()_ or _scrollYToPixel()_?

Thanks

**Comment by Jugen** [comment, marked as answer]
Call `area.requestFollowCaret();` after appending to scroll to bottom.

**Reply by noosphere888** [reply]
That works perfectly, thank you!
